### PR TITLE
Dockerfile: Added goss

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,11 +8,20 @@ ENV SIGNAL_BUILD_STOP=99 \
...
+    && \
+    # Add goss for local testing
+    curl -L https://github.com/aelsabbahy/goss/releases/download/v0.2.3/goss-linux-amd64 -o /usr/local/bin/goss && \
+    chmod +x /usr/local/bin/goss && \
+    apt-get remove --purge -yq curl && \
+    apt-get autoclean -y && \
     apt-get autoremove -y && \
     rm -rf /var/lib/{cache,log}/ && \
-    rm -rf /var/lib/apt/lists/ && \
+    rm -rf /var/lib/apt/lists/* && \
```
Comment (bryanlatten) [inline Dockerfile]: didn't intend to delete the folder, causes weird problems in 16.04 (vs. 14.04)

File: .travis.yml [hunk 2]
```diff
@@ -5,3 +5,4 @@ services:
 
 script:
 - docker build .
+- docker build -f Dockerfile-alpine .
```
Comment (bryanlatten) [inline .travis.yml]: now testing alpine as part of PR process

**Comment by apratina** [comment]
:+1:

**Comment by aivtsan** [comment]
👍

**Comment by bossjones** [comment]
👍

File: Dockerfile (hunk 1 above)
Comment (ko-be) [inline Dockerfile]: What about the /var/lib/cache & /var/lib/log dirs, are they ok to delete @bryanlatten ?

**Comment by ko-be** [comment]
One comment, otherwise 👍

File: Dockerfile (hunk 1 above)
Comment (bryanlatten) [inline Dockerfile]: @ko-be haven't been an issue thus far...

**Pull request merged**